crates/phactory/src/prpc_service.rs

```diff
@@ -412,13 +413,8 @@ impl<Platform: pal::Platform + Serialize + DeserializeOwned> Phactory<Platform>
 
                 cached_resp.attestation = Some(pb::Attestation {
                     version: 1,
-                    provider: "SGX".to_string(),
-                    payload: Some(pb::AttestationReport {
-                        report: attn_report,
-                        signature: base64::decode(sig).map_err(from_display)?,
-                        signing_cert: base64::decode_config(cert, base64::STANDARD)
-                            .map_err(from_display)?,
-                    }),
+                    provider: self.attestation_provider.clone(),
+                    payload: encoded_report,
```

Comment:
This is a breaking change which make a new version of `pherry` or `prb` won't talk to old `pruntime`s. Better create a new field, like `payload_v2`?